app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,43 @@
+<h2>Sign up</h2>
+
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+
+  <div class="field">
+    <%= f.label :name %><br />
+    <%= f.text_field :name %>
+  </div>
+
+  <div class="field">
+    <%= f.label :photo %><br />
+    <%= f.text_field :photo %>
+  </div>
+
+  <div class="field">
+    <%= f.label :bio %><br />
+    <%= f.text_field :bio %>
+  </div>
+
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password %>
+    <% if @minimum_password_length %>
+    <em>(<%= @minimum_password_length %> characters minimum)</em>
+    <% end %><br />
```

Comment:
- [ ] Kindly fix the error in this form, I failed to register on the app because of the same error. The same error prevented me from testing the app. See a screenshot below

![devise](https://user-images.githubusercontent.com/53124548/194499897-b7d2e90e-dced-46fa-aed2-aab04d01cb72.png)

Reply:
This change  has not yet been fixed

Reply:
@peterrobert But is working for me in my PC

Reply:
@peterrobert But it is working for me in my PC

Reply:
Try cloning the app from your github again and try to run it just to confirm